app/views/content/life-as-a-teacher/explore-subjects/history/_article.html.erb

```diff
@@ -0,0 +1,73 @@
+      <p>Your pupils will learn more from you than you might think. As their history teacher, you'll develop your pupils’ skills to investigate the past by interrogating primary and secondary sources. You'll teach them valuable life skills such as independent study, writing persuasively, and using evidence to back up points of view.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_history_jack' %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-history-programmes-of-study/national-curriculum-in-england-history-programmes-of-study">national curriculum for history</a>.</p>
+    <p>
+Key Stage 3 history aims to build on students' understanding of the history of Britain and the wider world. Some themes you may study include:</p>
+    <ul>
+      <li>the development of church, state and society</li>
+      <li>ideas, political power, industry and empire</li>
+      <li>challenges for Britain, Europe and the wider world 1901 to the present day</li>
+      <li>a local history study</li>
+      <li>an issue in world history</li>
+    </ul>
+    <p>In key stage 4 (14 to 16 years olds) you will help pupils demonstrate skills in:</p>
+    <ul>
+      <li>developing knowledge of key historical events</li>
+      <li>developing questioning through the  use of sources</li>
+      <li>analysing trends</li>
+      <li>creating structured accounts using evidence</li>
+    </ul>
+<p>Explore what teaching key stage 3 and 4 history lesson would be like at the <a href="https://www.thenational.academy/teachers/curriculum/history-secondary-aqa/units">Oak National Academy website</a>.</p>
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop further skills in:</p>
+    <ul>
+      <li>analysing sources</li>
+      <li>independent study</li>
+      <li>using evidence to support views</li>
+      <li>writing well structured arguments</li>
+    </ul>
+    <%= render 'content/shared/teaching/ages_abilities' %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading--box-blue">Start your journey into teaching history</h2>
+    <%= render 'content/shared/qualifications-training/check_qualifcations' %>
+    <p>History teacher training courses have had successful applications from candidates with a range of degrees, including politics, philosophy, archaeology and English.</p>
+    <%= render 'content/shared/qualifications-training/find_out_more' %>
+    <%= render 'content/shared/qualifications-training/non_uk_qualifications' %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <%= render 'content/shared/teaching/funding_support' %>
```

Comment:
I think you might need to create a new partial to mention tuition fee loans and maintenance loans (where the subject doesn't have a bursary/scholarship)